### PR TITLE
Added Password Manager

### PR DESCRIPTION
**Pull request by thevijayshankersharma**
Closes: #562 

- **Title** :  Password Manager
- **Name:** : Vijay Shanker Sharma
- **Idenitfy yourself:** : GSSOC

I added ```Password Manager```

### How Has This Been Tested? ⚙️

I tested it locally

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Properly follow and read the guidelines

**Comment by thevijayshankersharma** [comment]
@Avdhesh-Varshney Is it correct now ??

**Comment by thevijayshankersharma** [comment]
@Avdhesh-Varshney , is it correct now? If there are any other issues, please specify what needs to be done more specifically.

**Comment by meetarora10** [comment]
@thevijayshankersharma you have to enter your project in other README file .

**Comment by thevijayshankersharma** [comment]
@Avdhesh-Varshney @meetarora10 Done.

File: Vanilla-JS-Projects/Basic/Password-Manager/image.png
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Password-Manager/image.png:1]: change this ss from png to webp format. Name of ss file is `screenshot`

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Basic/Password-Manager/image.png
Comment (thevijayshankersharma) [reply Vanilla-JS-Projects/Basic/Password-Manager/image.png:1]: Sure.

Comment (thevijayshankersharma) [reply Vanilla-JS-Projects/Basic/Password-Manager/image.png:1]: @Avdhesh-Varshney Done. Could you check my others issues too.

File: Vanilla-JS-Projects/Basic/Password-Manager/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- Clone the repository or download the ZIP file.
+- Open `index.html` in your web browser.
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## 📷 Screenshots 📸
+
+<!-- Add screenshots or demo GIFs of your application -->
+
+![Screenshot](./image.png)
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Password-Manager/README.md:38]: .

File: Vanilla-JS-Projects/Basic/Password-Manager/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>Vijay Shanker Sharma</i></b> 👨‍💻</h4>
+<p align='center'>
+  <a href='https://www.linkedin.com/in/your-linkedin-profile/'>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Password-Manager/README.md:46]: .

File: Vanilla-JS-Projects/Basic/Password-Manager/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <img src='https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white' />
+  </a>
+  <a href='https://github.com/your-github-profile'>
```
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Basic/Password-Manager/README.md:49]: .

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Vanilla-JS-Projects/Basic/Password-Manager/README.md (hunk 1 above)
Comment (thevijayshankersharma) [reply Vanilla-JS-Projects/Basic/Password-Manager/README.md:38]: I will do this.

File: Vanilla-JS-Projects/Basic/Password-Manager/README.md (hunk 2 above)
Comment (thevijayshankersharma) [reply Vanilla-JS-Projects/Basic/Password-Manager/README.md:46]: what is wrong in this ?

File: Vanilla-JS-Projects/Basic/Password-Manager/README.md (hunk 3 above)
Comment (thevijayshankersharma) [reply Vanilla-JS-Projects/Basic/Password-Manager/README.md:49]: what is wrong in this ?

**Comment by thevijayshankersharma** [comment]
@Avdhesh-Varshney Done

**Review by Avdhesh-Varshney — APPROVED** [review]
Please fill this survey form https://forms.gle/Wvsp9N8TDF1WPStq9

**Pull request merged**